boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840.dts

```diff
@@ -188,7 +188,7 @@ arduino_i2c: &i2c0 {
 
 &spi1 {
 	compatible = "nordic,nrf-spi";
-	status = "okay";
+	status = "disabled";
```

Comment:
This shield uses `gpio1.8` (`miso-pin = <40>`), unless SPI1 is disabled it doesn't work. I'm not sure if there is a better way to solve that...

Reply:
I think the only way is to add `boards/shields/buydisplay_2_8_tft_touch_arduino/boards/nrf52840dk_nrf52840.overlay` and disable spi1 node there

Reply:
@jfischer-phytec-iot ok, it works. thanks for the suggestion!